core/src/wallet/bitcoin/api_impl/BitcoinLikeTransactionApi.cpp

```diff
@@ -280,29 +280,29 @@ namespace ledger {
                 //Var bytes Signature (prefix length)
                 //Get length of VarInt representing length of R
                 BytesWriter rLength;
-                rLength.writeVarInt(signatures[i].r.size());
+                rLength.writeByte(signatures[i].r.size());
                 //Get length of VarInt representing length of R
                 BytesWriter sLength;
```

Comment:
Now, as you know the size of length + size of data, you don't need this additional `BytesWriter`, the same applies for `sAndRLength`